app/lib/actions/katello/jail_concern/content_view.rb

```diff
@@ -0,0 +1,41 @@
+module Actions
+  module Katello
+    module JailConcern
+      module ContentView
+        def content_view_id
+          input['content_view']['id']
+        end
+
+        def content_view_name
+          input['content_view']['name']
+        end
+
+        def content_view_label
+          input['content_view']['label']
+        end
+
+        def self.included(base)
+          super
+          base.instance_eval do
+            apipie :class, "A class representing #{base} object" do
+              desc 'This object is available as **@object** variable in
+                    webhook templates when a corresponding event occures.
+                    The following properties can be used to retrieve the needed information.'
+              name "#{base}"
+              refs "#{base}"
+              sections only: %w[all webhooks]
+              property :task, object_of: 'Task', desc: 'Returns the task to which this action belongs'
+              property :content_view_id, Integer, desc: 'Returns the id of the content view'
+              property :content_view_name, String, desc: 'Returns the name of the content view'
+              property :content_view_label, String, desc: 'Returns the label of the content view'
+            end
+            jail_class = Class.new(::Actions::ObservableAction::Jail) do
+              allow :content_view_id, :content_view_name, :content_view_label
+            end
+            base.const_set('Jail', jail_class)
+          end
+        end
+      end
+    end
+  end
+end
```

Comment:
I think I found a bug tho, I am getting safemode error for Content View create event, I think this concern is not included in the ContentView model at all. It is not a Dynflow action but ActiveRecord event as well, so it should be just a simple Jail.

```
2021-04-06T12:24:34 [I|app|06a8fc9e] ForemanWebhooks::EventSubscriber: content_view_created.event.foreman event received
2021-04-06T12:24:34 [E|app|06a8fc9e] Safemode::NoMethodError: undefined method '#content_view_id' for Katello::ContentView::Jail (Katello::ContentView)
```

Reply:
Well, for content view object it should be just `id`, not the `content_view_id`. There is (or at least should be, I don't remember) documentation for Content View class in `all`/`additional` section in DSL docs.

Reply:
So it is used in other Dynflow action(s) but in Content View (CRUD) the Jail is missing, so actually here it is correct.